### PR TITLE
Document how cuDF is pronounced

### PR DESCRIPTION
**Pull request by pentschev**
## Description
Document in `README.md` and sphinx landing pages how cuDF is pronounced.

It is known people may pronounce cuDF in ways that aren't how it was conceived, such as "see-you-dee-ef". The correct way to pronounce is not documented anywhere so people who have never heard it from someone knowledgeable aren't able to know for sure, and thus this should be clearly documented. 

## Checklist
- [x] I am familiar with the [Contributing Guidelines](https://github.com/rapidsai/cudf/blob/HEAD/CONTRIBUTING.md).
- [x] New or existing tests cover these changes.
- [x] The documentation is up to date with these changes.

### CODE REVIEW

**Comment by pentschev** [comment]
I'm not entirely sure myself what is the right way to spell the pronounciation. I've chosen `cuh-dee-ef` because it seems closest to me, but I think other ways may be the correct choice instead:

- "cuh": maybe only "cu"
- "ef": maybe "eff" or "af" or "aff"

Hopefully someone can correct here if there's a better way to spell it.

**Comment by pentschev** [comment]
And also please let me know if there are more places where this should be documented.

**Comment by davidwendt** [comment]
I think the correct notation may be `kw - di - ef` since I believe it to be close to saying the letters `Q` `D` `F`

**Comment by davidwendt** [comment]
It their a pronunciation syntax for CUDA? I would take the first syllable from that.

File: docs/cudf/source/index.rst [hunk 1]
```diff
@@ -5,12 +5,12 @@ Welcome to the cuDF documentation!
     :width: 300px
     :align: center
 
-**cuDF** is a Python GPU DataFrame library (built on the `Apache Arrow
-<https://arrow.apache.org/>`_ columnar memory format) for loading, joining,
-aggregating, filtering, and otherwise manipulating data. cuDF also provides a
-pandas-like API that will be familiar to data engineers & data scientists, so
-they can use it to easily accelerate their workflows without going into
-the details of CUDA programming.
+**cuDF** (pronounced "cuh-dee-ef") is a Python GPU DataFrame library (built
```
Comment (bdice) [inline docs/cudf/source/index.rst:8]:
```suggestion
**cuDF** (pronounced "KOO dee-eff") is a Python GPU DataFrame library (built
```

File: docs/dask_cudf/source/index.rst [hunk 2]
```diff
@@ -6,8 +6,9 @@
 Welcome to dask-cudf's documentation!
 =====================================
 
-Dask-cuDF is an extension library for the `Dask <https://dask.org>`__
-parallel computing framework that provides a `cuDF
+**Dask-cuDF** (pronounced "desk-cuh-dee-ef") is an extension
```
Comment (bdice) [inline docs/dask_cudf/source/index.rst:9]: I would skip the pronunciation guide on the dask-cudf library. I think we can just do it for the main cuDF README. Also dask isn’t the same as desk in my pronunciation but I’m not enough of an expert to know how to write that word pronunciation properly.

**Review by bdice — CHANGES_REQUESTED** [review]

**Comment by pentschev** [comment]
> I think the correct notation may be `kw - di - ef` since I believe it to be close to saying the letters `Q` `D` `F`

I have to disagree with the `kw` part, I'm pretty sure CUDA is pronounced "koo-da" (already using @bdice 's suggestion, which I now agree with), you can listen to Jensen or anybody else at NVIDIA saying it.

Also with `kw-di-ef` it seems like you're trying to come closer to how that would be written in [IPA](https://en.wikipedia.org/wiki/Help:IPA/English) format, and although I think this has a very precise definition and is certainly a more consistent approach, I don't think the average person would know how to read/pronounce that (I know I wouldn't without researching first) and thus it would defeat the purpose of actually describing how to pronounce it in the first place, therefore I think we should try writing an "english pronunciation", or both "english pronunciation" + IPA, but not skip the english one. As an example, if `kw` 

> It their a pronunciation syntax for CUDA? I would take the first syllable from that.

The closest public thing I found is [this forum discussion from 2007](https://forums.developer.nvidia.com/t/cuda-pronunciation/846/4), but there are no official answers so I wouldn't blindly trust it.

File: README.md [hunk 3]
```diff
@@ -2,8 +2,9 @@
 
 ## 📢 cuDF can now be used as a no-code-change accelerator for pandas! To learn more, see [here](https://rapids.ai/cudf-pandas/)!
 
-cuDF is a GPU DataFrame library for loading joining, aggregating,
-filtering, and otherwise manipulating data. cuDF leverages
+cuDF (pronounced "cuh-dee-ef") is a GPU DataFrame library
```
[the remark this answers is not in the document]
Comment (pentschev) [reply README.md:5]: Agreed, I'll wait a couple more days with additional input before making those changes.

**Comment by bdice** [comment]
> It their a pronunciation syntax for CUDA? I would take the first syllable from that.

This is a great question. From your other comment, it sounded like there was a Q sound. I don’t pronounce CUDA with a Q sound, but some people do (I think Q is not the official pronunciation though). It turns out that “cu” making a koo sound is fairly rare. Cuckoo, cupola, and cuprous (to do with copper) are the examples I can find.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (davidwendt) [reply README.md:5]: I agree with the `KOO dee-eff` suggestion too.

[the remark this answers is not in the document]
Comment (ttnghia) [reply README.md:5]: I also support using `KOO` over `cuh`.

File: docs/dask_cudf/source/index.rst (hunk 2 above)
Comment (pentschev) [reply docs/dask_cudf/source/index.rst:9]: > I would skip the pronunciation guide on the dask-cudf library. I think we can just do it for the main cuDF README.

I wasn't sure if we should do it here either but decided in favor of this for the cases where people may end up directly here when looking to accelerate dataframes with Dask on the GPU.

> Also dask isn’t the same as desk in my pronunciation but I’m not enough of an expert to know how to write that word pronunciation properly.

From community meetings I've joined, "desk" is definitely more prevalent.

Comment (bdice) [reply docs/dask_cudf/source/index.rst:9]: ...Doesn't dask rhyme with task or mask, and not desk (or the first syllable of "pesky")? I think that dask is a self-explanatory pronunciation and I don't see a way to explain the pronunciation better than the word "dask" itself.

Comment (pentschev) [reply docs/dask_cudf/source/index.rst:9]: I gotta be honest and I always thought "mask", "task" and "desk" sounded the same (with the exception of the first letter, of course), then I went on to listen to them and can now grasp the (to me, subtle) difference. I agree with you that "dask" sounds like "dask" and not "desk".

With that said I'd say we can update here and say it's pronounced "dask-KOO-dee-eff". Once again, my primary purpose is to make sure people who land here as opposed to looking at cuDF by itself also know how to pronounce cuDF, Dask pronunciation was added just for completeness. If you still think we should remove it I can do that, but personally I feel there's a benefit to have it here too.

Comment (pentschev) [inline docs/dask_cudf/source/index.rst:9]:
```suggestion
**Dask-cuDF** (pronounced "DASK KOO-dee-eff") is an extension
```

File: docs/dask_cudf/source/index.rst [hunk 4]
```diff
@@ -6,8 +6,9 @@
 Welcome to dask-cudf's documentation!
 =====================================
 
-Dask-cuDF is an extension library for the `Dask <https://dask.org>`__
-parallel computing framework that provides a `cuDF
+**Dask-cuDF** (pronounced "DASK KOO-dee-eff") is an extension
```
Comment (bdice) [inline docs/dask_cudf/source/index.rst:9]: Small request: we have a dash in `KOO-dee-eff` but not the other pronunciation guides (`KOO dee-eff`). Should we remove it here, or add it in the other two? I'm fine with either option.

**Comment by pentschev** [comment]
@bdice @davidwendt @ttnghia all comments so far have been addressed, are there anything else or are you good if we merge it?

**Review by davidwendt — APPROVED** [review]

**Review by ttnghia — APPROVED** [review]

**Review by bdice — APPROVED** [review]

**Pull request merged**